util/setup_base.js

```diff
@@ -40,60 +36,49 @@ const config = {
     // This is your permission level, the staff levels should always be above the rest of the roles.
     { level: 2,
       // This is the name of the role.
-      name: "Moderator",
+      name: 'Trainee',
       // The following lines check the guild the message came from for the roles.
       // Then it checks if the member that authored the message has the role.
       // If they do return true, which will allow them to execute the command in question.
       // If they don't then return false, which will prevent them from executing the command.
       check: (message) => {
         try {
-          const modRole = message.guild.roles.find(r => r.name.toLowerCase() === message.settings.modRole.toLowerCase());
-          if (modRole && message.member.roles.has(modRole.id)) return true;
+          const config = require('./config.js');
+          if (message.member.roles.has(config.traineeRole)) return true;
         } catch (e) {
           return false;
         }
       }
     },
 
     { level: 3,
-      name: "Administrator", 
+      name: 'Council', 
       check: (message) => {
         try {
-          const adminRole = message.guild.roles.find(r => r.name.toLowerCase() === message.settings.adminRole.toLowerCase());
-          return (adminRole && message.member.roles.has(adminRole.id));
+          const config = require('./config.js');
+          if (message.member.roles.has(config.councilRole)) return true;
         } catch (e) {
           return false;
         }
       }
     },
-    // This is the server owner.
-    { level: 4,
-      name: "Server Owner", 
-      // Simple check, if the guild owner id matches the message author's ID, then it will return true.
-      // Otherwise it will return false.
-      check: (message) => message.channel.type === "text" ? (message.guild.owner.user.id === message.author.id ? true : false) : false
-    },
-
-    // Bot Support is a special inbetween level that has the equivalent of server owner access
-    // to any server they joins, in order to help troubleshoot the bot on behalf of owners.
-    { level: 8,
-      name: "Bot Support",
-      // The check is by reading if an ID is part of this array. Yes, this means you need to
-      // change this and reboot the bot to add a support user. Make it better yourself!
-      check: (message) => config.support.includes(message.author.id)
-    },
-
     // Bot Admin has some limited access like rebooting the bot or reloading commands.
     { level: 9,
-      name: "Bot Admin",
-      check: (message) => config.admins.includes(message.author.id)
+      name: 'Police', 
+      check: (message) => {
+        try {
+          const config = require('./config.js');
+          if (message.member.roles.has(config.policeRole)) return true;
```

Comment:
`return message.member.roles.has(config.policeRole);`

Reply:
Also, this try/catch block could be redundant.

Reply:
Should be good to go.